config/locales/en.yml

```diff
@@ -267,22 +267,31 @@ en:
     smod_help_text: Regulation 38 (Civil Legal Aid (Financial Resources and Payment for Services) Regulations 2013) provides for a disregard of the amount or value of assets that are the subject matter of dispute (SMOD); however the total amount disregarded must not exceed £100,000. Please read the related content on subject matter of dispute (Regulation 38).
   estimate_flow:
     partner:
+      partner_guidance:
+        text: "Guidance on partners"
+        link: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1045682/Lord_Chancellor_s_guidance_on.pdf#page=17
       caption: About your client
       legend: Does your client have a partner?
       hint: This is a spouse, someone with whom they are in a civil relationship, a cohabiter or someone they share their finances with. They cannot be the opponent in any legal proceedings.
     applicant:
+      passporting_guidance:
+        text: "Guidance on Passporting"
```

Comment:
With this and the pensioner guidance again I would be surprised if adding a capital P in the middle was a deliberate, convention-breaking choice. I'd confirm with Chris.